extension/app/manifest.json

```diff
@@ -17,13 +17,15 @@
   },
   "permissions": [
     "activeTab",
-    "debugger"
+    "debugger",
+    "tabs"
   ],
   "browser_action": {
     "default_icon": {
       "38": "images/icon-38.png"
     },
     "default_title": "Lighthouse",
     "default_popup": "popup.html"
-  }
+  },
+  "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'"
```

Comment:
can we set `object-src` to `none`? And eval is a bummer here :( I don't follow the latest in templating at all, though, so I trust you on handlebars